### ISSUE TITLE
ESLint 4: support module `eslint-config-standard`

### ISSUE THREAD

**Issue by chuyik** [codeclimate/codeclimate-eslint#348]
ESLint 4 was reported: 
```
Module not supported: eslint-config-standard
ESLint is running with the default parser.
```

Could this plugin be supported on ESLint 4?

**Comment by toddmazierski** [comment]
Hi, @chuyik!

It doesn't appear as though this plugin is fully ESLint 4, but it's being looked into: https://github.com/standard/eslint-config-standard/issues/82

**Comment by wingrunr21** [comment]
Can this be revisited at this time?

**Comment by filipesperandio** [comment]
Seems like this was added to the plugin late last year. Closing.
